modules/draco/src/draco-loader.js

```diff
@@ -17,7 +17,8 @@ export const DracoWorkerLoader = {
     draco: {
       decoderType: typeof WebAssembly === 'object' ? 'wasm' : 'js', // 'js' for IE11
       libraryPath: `libs/`,
-      workerUrl: `https://unpkg.com/@loaders.gl/draco@${VERSION}/dist/draco-loader.worker.js`
+      workerUrl: `https://unpkg.com/@loaders.gl/draco@${VERSION}/dist/draco-loader.worker.js`,
+      localWorkerUrl: `modules/draco/dist/draco-loader.worker.js`
```

Comment:
this requires the application specifies the correct `contentBase` in the `babel config` file.

Reply:
Yes, good idea you can update in the debug section at the bottom of the doc I created. 

At least for application debugging that would work (although it could break other things being served). 

Deployment with local workers is a different story. 

To handle those cases, it could be better to have a global option `localWorkerPath`. But one thing at a time...